src/lib/dlt_user.c

```diff
@@ -2799,7 +2799,10 @@ void dlt_user_trace_network_segmented_thread(void *unused)
         if (read < 0) {
             if (errno != EINTR) {
                 dlt_vlog(LOG_WARNING, "NWTSegmented: Error while reading queue: %s\n", strerror(errno));
-                usleep(DLT_USER_MQ_ERROR_RETRY_INTERVAL);
+                struct timespec req;
+                req.tv_sec = 0;
+                req.tv_nsec = DLT_USER_MQ_ERROR_RETRY_INTERVAL * 1000;
```

Comment:
As user might change the value of DLT_USER_MQ_ERROR_RETRY_INTERVAL to more than 1sec, this needs to be taken care.

Reply:
Do  convert